### DISCUSSION TITLE
Error with magic links

### DISCUSSION THREAD

**Discussion by talhabw** [supabase/supabase#3045] [Questions] [answered]
Hi. I am trying to follow Vue 3 docs to implement supabase to my project. i have a problem with auth.
i am handling login in my login page.
magic link successfully comes to my inbox.
but when i open it
it redirects me to my project with an error in url:
`http://localhost:3000/admin/#error_code=404&error_description=User+not+found`
why is this happening?

i can see the user in the app.supabase.io

and every time i opened magic link it updates the 'last sign in'

i have this code to handle login at /admin
```
supabase.auth.onAuthStateChange((_, session) => {
  store.commit("UPDATE_USER", { session })
}) 
```

**Comment by kangmingtay** [comment]
Hey @talhabw, hmm, im not too sure how you're calling the magiclink endpoint. Are you using the `signIn` or `signUp` methods? Also, did you set the redirect url on the dashboard to redirect to your admin endpoint?

**Reply by talhabw** [reply]
this is my login function:
```js
const handleLogin = async () => {
  try {
    loading.value = true
    const { error } = await supabase.auth.signIn({
      email: email.value,
      options: {
        redirectTo: "http://localhost:3000/admin",
      },
    })
    if (error) throw error
    alert("Check your email for the login link!")
  } catch (error) {
    alert(error.error_description || error.message)
  } finally {
    loading.value = false
  }
}
```
I set the redirect URL on the dashboard to `http://localhost:3000/admin`. But magic link still redirects to `http://localhost:3000`. I changed the redirectTo parameter in the URL manually, but I'm still getting the same error.

**Comment by talhabw** [comment, marked as answer]
apparently, it was an issue unrelated to supabase.
